core/controllers/topics_and_skills_dashboard_test.py

```diff
@@ -216,3 +248,33 @@ def test_skill_creation_in_valid_topic(self):
                 topic.uncategorized_skill_ids,
                 [self.linked_skill_id, skill_id])
             self.logout()
+
+
+class MergeSkillHandlerTest(BaseTopicsAndSkillsDashboardTest):
+
+    def setUp(self):
+        super(MergeSkillHandlerTest, self).setUp()
+        self.url = feconf.MERGE_SKILL_URL
+
+    def test_merge_skill(self):
```

Comment:
Can you also check if questions linked to old skill are also updated? By maybe linking some sample questions to the old skill and querying their corresponding QuestionSkillLink models after updation?

Reply:
Done